enginetest/memory_engine_test.go

```diff
@@ -100,34 +100,38 @@ func TestSingleScript(t *testing.T) {
 
 	var test enginetest.ScriptTest
 	test = enginetest.ScriptTest{
-		Name: "multiple triggers before and after insert, with precedes / follows",
+		Name: "trigger before insert, multiple triggers defined",
```

Comment:
this should go in `enginetest/trigger_queries.go`

Reply:
It is, I just copy and paste tests there when I want to run them in isolation. It causes churn in this file, but I'm not super concerned.